Deploy/stacks/dynamic/stack-clients/src/main/java/com/cmclinnovations/stack/clients/geoserver/GeoServerClient.java

```diff
@@ -107,6 +112,43 @@ public void loadStyle(GeoServerStyle style, String workspaceName) {
         }
     }
 
+    public void loadOtherFiles(Path baseDirectory, List<GeoserverOtherStaticFile> files) {
+        files.forEach(file -> {
+            loadStaticFile(baseDirectory, file);
+        });
+
+    }
+
+    private void loadStaticFile(Path baseDirectory, GeoserverOtherStaticFile file) {
+        Path filePath = baseDirectory.resolve(file.getSource());
+        String sourceParentDir = filePath.getParent().toString();
+        String fileName = filePath.getFileName().toString();
+        String absTargetDir = Path.of(STATIC_DATA_DIRECTORY).resolve(Path.of(file.getTarget())).toString();
```

Comment:
See comment above. Also I'm pretty sure you can ```resolve``` Strings directly.
```suggestion
        Path absTargetDir = STATIC_DATA_DIRECTORY.resolve(file.getTarget());
```